lang/cs/Org.Apache.REEF.Bridge.Client/GrpcGeneratedCode/DriverClientProtocolGrpc.cs

```diff
@@ -0,0 +1,854 @@
+// <auto-generated>
+//     Generated by the protocol buffer compiler.  DO NOT EDIT!
+//     source: DriverClientProtocol.proto
+// </auto-generated>
```

Comment:
Interestingly, there is a different header for the four auto-generated file any idea why?

Reply:
Checking

Reply:
I am adding Apache copyrights as part of the build process. This is necessary or the rat check will fail.